test/custom-header.spec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const expect = require('chai').expect;
+
+module.exports = (fetchMock, theGlobal, Headers) => {
+
+    describe('fetch-mock', () => {
+
+        const dummyFetch = () => Promise.resolve(arguments);
+
+        before(() => {
+            theGlobal.fetch = dummyFetch;
+        })
+
+        describe('Interface', () => {
+
+            it('handles Headers class configuration correctly', () => {
+                fetchMock.mock({
+                    name: 'route1',
+                    headers: {
+                        test: 'yes'
+                    },
+                    matcher: 'http://it.at.here/',
+                    response: 'ok'
+                }).catch();
+                return Promise.all([
+                    fetch('http://it.at.here/', {headers: {test: 'yes'}}),
+                    fetch('http://it.at.here/', {headers: new Headers({test: 'yes'})}),
```

Comment:
**Case 1**

In one of our projects we're using fetch with the `Headers` class from [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch). 

_Example API call_
```javascript_
fetch('/api', {
    'method': 'POST',
    'headers': new Headers({'X-CHUCK': 'NORRIS'}),
})
```

_Example test call_
```javascript
fetchMock.postOnce('/api', new Response(stubResponse, { status: 200 }), {headers: {'x-chuck': 'norris'}});
```

Without the third parameter, this mock will match and the test succeeds. But passing the custom headers param will result in an **unmatched api call**.
Internally, the `X-CHUCK` header is stored as array:
```javascript
console.log(fetchMock.lastCall()[1].headers._headers);
// { 'x-chuck': ['NORRIS'] }
```

**Question**: Is there any way using the `Headers` class that way? What's your suggestion on it?